src/overview/copy-paster/background/index.ts

```diff
@@ -0,0 +1,62 @@
+import Storex from '@worldbrain/storex'
+import { bindMethod } from 'src/util/functions'
+import { BrowserSettingsStore } from 'src/util/settings'
+import CopyPasterStorage from './storage'
+import { RemoteCopyPasterInterface } from './types'
+
+export default class CopyPasterBackground {
+    storage: CopyPasterStorage
+    remoteFunctions: RemoteCopyPasterInterface
+
+    constructor(
+        private options: {
+            storageManager: Storex
+        },
+    ) {
+        // makes the custom copy paster table in indexed DB
+        this.storage = new CopyPasterStorage({
+            storageManager: options.storageManager,
+        })
+
+        this.remoteFunctions = {
+            createTemplate: bindMethod(this, 'createTemplate'),
+            findTemplate: bindMethod(this, 'findTemplate'),
+            updateTemplate: bindMethod(this, 'updateTemplate'),
+            deleteTemplate: bindMethod(this, 'deleteTemplate'),
+            findAllTemplates: bindMethod(this, 'findAllTemplates'),
+        }
+    }
+
+    async createTemplate({ title, code, isFavourite = false }) {
```

Comment:
Could you please add full types to these signatures? What I normally do to prevent redundancy is write it as:
```
createTemplate(params: { title: string, code: string, isFavourite?: boolean }) 
```